### PR TITLE
Adding DNA simulations

### PR DESCRIPTION
**Pull request by rbharath**
This PR adds support for DNA simulations by importing `simulations.py` as used by DRAGONN (see #1002). However to do this, the PR has to add a DeepChem dependence on https://github.com/kundajelab/simdna.  Unlike our other dependencies, simdna is primarily research code. However, I think with the version pinned as here, it should be relatively stable.

WIP until all tests pass.
 
Edit: Now using the brand new pip version for `simdna`.

### CODE REVIEW

**Comment by rbharath** [comment]
Unfortunately, `simdna` has some Python 3.5 issues. The Travis CI build on 3.5 is failing with this error https://github.com/kundajelab/simdna/issues/3.

Will think about a good way to deal with this error.

File: scripts/install_deepchem_conda.sh [hunk 1]
```diff
@@ -38,3 +38,9 @@ conda install -y -q -c conda-forge zlib=1.2.11
 conda install -y -q -c conda-forge requests=2.18.4
 conda install -y -q -c conda-forge xgboost=0.6a2
 conda install -y -q -c rdkit rdkit=2017.09.1
+pushd .
+cd $HOME
+git clone https://github.com/deepchem/simdna
```
Comment (lilleswing) [inline scripts/install_deepchem_conda.sh]: We should install a pinned version from our DeepChem channel.  Do you have a preferred git hash?

Comment (rbharath) [reply scripts/install_deepchem_conda.sh]: Good suggestion, fixed.

**Comment by rbharath** [comment]
I think this PR is ready to merge once tests pass. More work is needed to improve Dragonn integration and better support DNA simulations, but that's better left for a different PR.

**Comment by rbharath** [comment]
Unfortunately ran into a further issue to resolve with `pip` (https://github.com/kundajelab/simdna/issues/6), so back to WIP till that's worked out.

**Comment by rbharath** [comment]
The `pip` installation issues have been resolved upstream in `simdna` so going to go ahead and merge this PR in.

**Pull request merged**